ui/apps/platform/src/Containers/VulnMgmt/List/Cves/ListCVEs.utils.js

```diff
@@ -18,7 +18,9 @@ export function getFilteredCVEColumns(columns, workflowState) {
 
     const currentEntityType = workflowState.getCurrentEntityType();
 
-    // No need to show entities in the node component context.
+    const shouldKeepDiscoveredTime = currentEntityType !== entityTypes.CLUSTER_CVE;
```

Comment:
The `createdAt` field is always `null` for ClusterVulnerability, I guess because we don't actually scan the orchestrator, but instead just display all known vulns for a version of it. So, that column will always be blank. (If it becomes available later for some reason, we can just back this out.)